dallinger/command_line.py

```diff
@@ -277,16 +286,13 @@ def debug(verbose, bot):
     logfile = config.get('logfile')
     if logfile != '-':
         logfile = os.path.join(cwd, logfile)
-    config.extend({
-        "mode": u"debug",
-        "loglevel": 0,
-        "logfile": logfile
```

Comment:
I've addressed this oversight, by updating the logfile if it was set. See below.